### DISCUSSION TITLE
Building tesla-anderoid using Mac?

### DISCUSSION THREAD

**Discussion by adamChrystie** [tesla-android/issue-tracker#169] [Q&A]
Hi, anyone tried building tesla-android? I have a Mac with Windows 10 via boot camp. Any tips such as best to use a  Virtual machine on Mac  running Ubuntu  or perhaps better to use boot camp on Mac to load windows and run Ubuntu from a vm? Anyone triple booting on Mac in order to avoid using virtual machine. For example, Mac OS, Ubuntu & windows (boot camp ) ? 

I was hoping to tinker a bit and see if I can help address bugs in future. At least would be good learning experience.

**Comment by adamChrystie** [comment]
Well I made a Ubuntu VM on my Mac. 24GB of disk space was not enough. Time for me to better determine how how much disk space I'll need :).

**Reply by mikegapinski** [reply]
You need like 500gb to build this 😅

**Reply by mikegapinski** [reply]
One build takes around 6 hours on my setup 😅

**Reply by adamChrystie** [reply]
Alrighty, remade my VM to have 800GB of disk space. 60% done downloading from GitHub. Soon the iMac will be tortured with this long build ;).

**Reply by mikegapinski** [reply]
Just FYI, this android base is getting replaced very soon with a more generic one that will run on both rpi and my own device. Soo, this download will be only good for a release or two 😅
